JMeterAKSPipeline/DeleteLoadTestInfra.ps1

```diff
@@ -0,0 +1,41 @@
+param(
+    [Parameter(Mandatory = $true)]
+    [string]$aksClusterName,
+    [Parameter(Mandatory = $true)]
+    [string]$resourceGroup,
+    [Parameter(Mandatory = $True)]
```

Comment:
Can we make $true as consistent. some place we are using pascal casing $True.

Reply:
Done